frontend/src/components/PasswordInput.tsx

```diff
@@ -1,9 +1,17 @@
-import { FormControl, FormLabel, Input } from '@chakra-ui/react'
+import { FormControl, FormLabel, Input } from '@chakra-ui/react';
 
-const PasswordInput = () =>
+interface Props {
```

Comment:
Vi kan fjerne passwordInput, og omdøbe usernameinput, noget mere favnene som InputFormControl, som så kan bruges som komponent på både signup og login